content/docs/dvclive/usage-with-dvc.md

```diff
@@ -0,0 +1,125 @@
+# dvclive with DVC
+
+Even though dvclive does not require DVC to function properly, it includes a lot
+of integrations with DVC that user might find valuable. In this section we will
+modify the [example from previous one](/doc/dvclive/usage) to see how DVC can
+cooperate with dvclive.
+
+Let's use the code prepared in previous example and try to make it work with
+dvc. Training file `train.py` content:
+
+```python
```

Comment:
I mentioned that we might want to link to https://github.com/iterative/dvc-checkpoints-mnist, but alternatively you could incorporate checkpoints in this example. That might be too much to dive right into, but I think most use cases for dvclive with dvc will include checkpoints.

Reply:
Then maybe we need one more section? We integrated the project with DVC and dvclive, lets make some experiments?

Reply:
I think the changes to introduce checkpoints would be to write the model out in the callback, and to add the model output to dvc with `stage add --checkpoint model ...`. I don't see checkpoints as necessarily being tied to experiments. It adds versioning of the model output that aligns with the metrics dvclive is tracking.

However, as you mentioned below, let's think about how we can introduce multiple examples to cover all of this. I think the example you have is a great starting point.